pkg/monitor/api.go

```diff
@@ -60,6 +61,7 @@ func Start(ctx context.Context, restConfig *rest.Config, additionalEventInterval
 	startPodMonitoring(ctx, m, client)
 	startNodeMonitoring(ctx, m, client)
 	startEventMonitoring(ctx, m, client)
+	shutdown.StartMonitoringGracefulShutdownEvents(ctx, m, client)
```

Comment:
All the upgrade presubmits look unhealthy in an unusual way, is it this change blocking something or just bad luck?

Reply:
i had successful runs before, i pushed some changes after that, so maybe my changes or just bad luck, investigating...

Reply:
>  is it this change blocking something
yes, i didn't realize that, i just fixed it and pushed new changes